### ISSUE TITLE
Is it possible to ingest logs, traces, and metrics in OTLP format?

### ISSUE THREAD

**Issue by keisukesakasai** [openobserve/openobserve-docs#7]
Hi 👋

It seems like OTLP/gRPC and OTLP/HTTP are supported as of v0.6.0.
https://github.com/openobserve/openobserve/releases/tag/v0.6.0

However, in the documentation, I noticed that OTLP is not yet listed under ingestion. Is an update on this coming soon?
https://openobserve.ai/docs/ingestion/

**Comment by prabhatsharma** [comment]
Yes, it's possible to use OTLP (gRPC and HTTP). However, ingestion menu does not have the details. We are releasing 0.7.0 today, which will have OTLP details in the ingestion menu.

**Comment by keisukesakasai** [comment]
Thanks for the reply. Congrats on the release!
